tests/app/test_session.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from flask import url_for
+
+
+@pytest.mark.usefixtures("app")
+@pytest.mark.parametrize(
+    "todo, expected",
+    [
+        (
+            True,
+            "/sign-out?todo=True",
+        ),
+        (
+            False,
+            "/sign-out?todo=False",
+        ),
+    ],
+)
+def test_url(todo, expected):
+    url = url_for("session.get_sign_out", todo=todo)
+    assert url == expected
+
+
+@pytest.mark.usefixtures("app")
+def test_url_no_args():
```

Comment:
I'm not convinced, we need this. This is testing flask's built in `url_for` method. This would work regardless of if the route handled query params. Since this is a route, the scenario should be tested under integrations/functional tests.

Reply:
Yeah, I was not convinced but wanted to try to test, I was expecting you will want to scrap it as we don't do any similar test.  I'll remove it now.